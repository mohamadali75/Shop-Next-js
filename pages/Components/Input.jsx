import React from "react";
import { css } from "@emotion/css";
import { useTheme } from "@mui/material";



function Input({type,placeholder,className}) {
    const theme=useTheme()
    return(
        <input className={css`
        border: solid ${theme.palette.primary.border};
        
   }
   }
 `}  type={type} placeholder={placeholder}/>
    )
}
export default Input
