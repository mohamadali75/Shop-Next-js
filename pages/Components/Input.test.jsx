import renderer from 'react-test-renderer';
import React from "react"
import Input from './Input';

it('Input test', () => {
  const component = renderer.create(
    <Input>Test</Input>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();})
