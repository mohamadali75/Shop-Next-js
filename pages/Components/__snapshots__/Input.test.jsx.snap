// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input test 1`] = `
<input
  className="css-10s24d6"
/>
`;
